src/main/java/org/glassfish/copyright/AbstractCopyright.java

```diff
@@ -93,6 +95,8 @@
     private static Pattern bsdpat = Pattern.compile(
 	"(THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS)"+
 	"|(SPDX-License-Identifier: BSD-3-Clause)", Pattern.MULTILINE);
+	// patter to detect DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
```

Comment:
nit typo:  "pattern"

Reply:
fixed in https://github.com/eclipse-ee4j/glassfish-copyright-plugin/commit/620fdb207832e090dba2f0ac01aa3db89af8554e